app_preemptible.py

```diff
@@ -0,0 +1,39 @@
+# !pip install 'git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git'
+import lightning as L
+from diffusion_with_autoscaler import BatchText, Text, PreemptibleRollout, AutoScaler
+
+
+class MyPythonServer(L.app.components.PythonServer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            input_type=BatchText,
+            output_type=BatchText,
+            *args,
+            **kwargs,
+        )
+
+    def setup(self):
+        pass
+
+    def predict(self, requests):
+        texts = [request.text for request in requests.inputs]
+        return BatchText(inputs=[{"text": text} for text in texts])
+
+
+component = AutoScaler(
+    MyPythonServer,
+    cloud_compute=L.CloudCompute("gpu-rtx", disk_size=80),
+    strategy=PreemptibleRollout(interval=7),
```

Comment:
Let's call it `IntervalReplacement`. What do you think @edenlightning @carmocca @awaelchli ?

Reply:
I don't have much context here but `ReplacementInterval` is easier to read than the other way around